person.rb

```diff
@@ -1,14 +1,31 @@
-class Person
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+require_relative 'book'
+require_relative 'Rental'
+require 'date'
+
+class Person < Nameable
   def initialize(age, name = 'unknown', parent_permission: true)
+    super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def correct_name
+    @name
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] in the person class you should also be able to change the rentals from outside the class, please add it therefor to the attr_accessor.